docs/documentation/server_admin/topics/threat/brute-force.adoc

```diff
@@ -74,7 +75,8 @@ When {project_name} disables a user, the user cannot log in until an administrat
 |15 minutes.
 
 |Failure Reset Time
-|The time when the failure count resets. The timer runs from the last failed login.
+|The time when the failure count resets. The timer runs from the last failed login. Make sure this number is always greater than `Max wait`, otherwise the effective
```

Comment:
```suggestion
|The time when the failure count resets. The timer runs from the last failed login. Make sure this number is always greater than `Max wait`; otherwise the effective
```